src/widget/wtracktableview.cpp

```diff
@@ -1167,7 +1167,8 @@ void WTrackTableView::dropEvent(QDropEvent * event) {
 
         QList<QString> fileLocationList;
         foreach (const QFileInfo& fileInfo, fileList) {
-            fileLocationList.append(fileInfo.canonicalFilePath());
+            // TODO(uklotzde): Replace with TrackRef::location()
```

Comment:
The corresponding commit is included in my "Write Audio Tags" PR that I try 
to submit step by step ;)

On 02/17/2016 10:08 PM, Sean M. Pappalardo wrote:
>
> In src/widget/wtracktableview.cpp 
> <https://github.com/mixxxdj/mixxx/pull/896#discussion_r53231494>:
>
> > @@ -1167,7 +1167,8 @@ void WTrackTableView::dropEvent(QDropEvent * event) {
> >
> >          QList<QString> fileLocationList;
> >          foreach (const QFileInfo& fileInfo, fileList) {
> > -            fileLocationList.append(fileInfo.canonicalFilePath());
> > +            // TODO(uklotzde): Replace with TrackRef::location()
>
> Why not take care of this TODO now? How much work is it?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/896/files#r53231494>.
>